Project 8 - Mapty/js/script.js

```diff
@@ -0,0 +1,294 @@
+"use strict";
+
+const form = document.querySelector(".form");
+const containerWorkouts = document.querySelector(".workouts");
+const inputType = document.querySelector(".form__input--type");
+const inputDistance = document.querySelector(".form__input--distance");
+const inputDuration = document.querySelector(".form__input--duration");
+const inputCadence = document.querySelector(".form__input--cadence");
+const inputElevation = document.querySelector(".form__input--elevation");
+
+class Workout {
+  date = new Date();
+  id = (Date.now() + "").slice(-10);
+
+  constructor(coords, distance, duration) {
+    this.coords = coords; //[lat, lng]
+    this.distance = distance; //in km
+    this.duration = duration; //in min
+  }
+
+  _setDescription() {
+    // prettier-ignore
```

Comment:
@sknepali Can't we avoid this ignore?

Reply:
Yes, We can avoid this. I used this because to display months array in a single line. Prettier makes it appear in mulitple lines. This will help to reduce extra line and in my openion, here for this months array its seems good in one line.